output/eu_crude/analysis.py

```diff
@@ -74,8 +78,148 @@ def get_waste_id(resource_array):
     return set(wasteid)
 
 
+def get_colors():
+    """ Gets colors from a matplotlib dictionary
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    Colores: array
+        array of color hex codes
+    """
+
+    colores = []
+    for hex in matplotlib.colors.cnames.items():
+        colores.append(hex[1])
+    return colores
+
+
+def get_power(filename):
+    """ Gets capacity vs time for every country
+        in stacked bar chart.
+
+    Parameters
+    ----------
+    filename: file
+        cyclus output file used
+
+    Returns
+    -------
+    plot of net capapcity vs time
+
+    """
+
+    sim_time = int(cur.execute('select endtime from finish').fetchone()[0]) + 1
+    timestep = np.linspace(0, sim_time, num=sim_time + 1)
+    powercap = []
+    reactor_num = []
+    countries = []
+    # get power cap values
+
+    governments = cur.execute('sELECT prototype, agentid FROM agententry\
+                              WHERE spec = ":cycamore:DeployInst"').fetchall()
+
+    entry = cur.execute('sELECT power_cap, agententry.agentid, parentid, entertime\
+                        FROM agententry INNER JOIN\
+                        agentstate_cycamore_reactorinfo\
+                        ON agententry.agentid =\
+                        agentstate_cycamore_reactorinfo.agentid').fetchall()
+
+    exit = cur.execute('sELECT power_cap, agentexit.agentid, parentid, exittime\
```

Comment:
oh for this part, exit has two innerjoins ( because the agentexit table has only agentid and exittime)
to get power_cap (from reactorinfo) and parentid (from agententry)

So doing so with re.sub would be difficult..?

It seems like it would be most ideal if edits were made to cyclus so that:
1. agentexit has as many columns (or just add parentid) as agententry
2. agentstate_cycamore_reactorinfo has the decommissioned reactors.

I will also post this to issue #22